ci/build_setup.ps1

```diff
@@ -0,0 +1,22 @@
+$ErrorActionPreference = "Stop";
+trap { $host.SetShouldExit(1) }
+
+if ("$env:NUM_CPUS" -eq "") {
+  $env:NUM_CPUS = (Get-WmiObject -class Win32_computersystem).NumberOfLogicalProcessors
+}
+
+if ("$env:ENVOY_BAZEL_ROOT" -eq "") {
+  Write-Host "ENVOY_BAZEL_ROOT must be set!"
+  throw
+}
+
+mkdir -force "$env:ENVOY_BAZEL_ROOT" > $nul
+
+$env:ENVOY_SRCDIR = [System.IO.Path]::GetFullPath("$PSScriptRoot\..")
+
+echo "ENVOY_BAZEL_ROOT: $env:ENVOY_BAZEL_ROOT"
+echo "ENVOY_SRCDIR: $env:ENVOY_SRCDIR"
+
+$env:BAZEL_BASE_OPTIONS="--nomaster_bazelrc --output_base=$env:ENVOY_BAZEL_ROOT --bazelrc=$env:ENVOY_SRCDIR\windows\tools\bazel.rc --batch"
```

Comment:
`--batch` is deprecated. You'll have to manually shut down the Bazel server afterwards.